modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -427,6 +443,36 @@ void ONNXImporter::populateNet(Net dstNet)
                 layerParams.set("has_bias", false);
             }
         }
+        else if (layer_type == "BatchNormalization")
```

Comment:
It looks like a merging artifact. There are two "BatchNormalization" in this file now.

Reply:
done